lib/puppet/pops/types/puppet_object.rb

```diff
@@ -11,6 +11,16 @@ module PuppetObject
   def _ptype
     self.class._ptype
   end
+
+  def _pall_contents(path, &block)
+  end
+
+  def _pcontents
+  end
+
+  def i12n_hash
```

Comment:
Yes, on an instance, this should be named "_pi12n_hash". The term "i12n_hash" is used throughout in the Pcore type system. It's rather late to change that now.